test/loading.jl

```diff
@@ -86,6 +86,18 @@ import UUIDs: UUID, uuid4, uuid_version
 import Random: shuffle, randstring
 using Test
 
+shastr = "ab"^20
+hash = SHA1(shastr)
+@test hash == eval(parse(repr(hash))) # check show method
+@test string(hash) == shastr
+@test "check $hash" == "check $shastr"
```

Comment:
The last two tests seem redundant because interpolation just calls `string`

Reply:
What?  I just looked at `@code_lowered` for `"foo $x"`, and it lowers to `string("foo ", x)`, which calls `print_to_string`, which calls `print` (not `show`).

What am I missing?

Reply:
The documentation of `string(xs...)` also says that it calls `print`.   Something seems wrong here.